test/jasmine/tests/layout_images_test.js

```diff
@@ -107,7 +107,7 @@ describe('Layout images', function() {
         it('should draw images on the right layers', function() {
 
             Plotly.plot(gd, data, { images: [{
-                source: 'imageabove',
```

Comment:
These were generating `404` warnings when you run the tests, which seem distracting and necessary, so I just put in `jsLogo` everywhere.